docs/client-provisioning-methods.adoc

```diff
@@ -0,0 +1,40 @@
+// Copy of this topic: https://docs.ota.here.com/client-config/client-provisioning-methods.html adapted for aktualizr repo. Replaces duplicate "automatic-provisioning.adoc"
+
+In OTA Connect, the provisioning process ensures that the device has a certificate and a unique identifier. After a device has been provisioned, it can receive software updates. OTA Connect supports two types of provisioning:
+
+* *Provisioning with Shared Credentials*
+
+Devices use temporary provisioning credentials to request permanent credentials from the server. You download the temporary provisioning credentials from the OTA Connect web app and install them on a device. Once the device registers with the server for the first time, the server assigns permanent credentials to the device. The device then uses these permanent credentials for all future transactions.
+
+
+* *Provisioning with Device Credentials*
++
+Devices already have permanent credentials installed. The server doesn't issue any credentials to devices. Instead, you use a root CA certificate to sign the credentials that you install on the device. You then install the same root CA certificate on the OTA Connect server.
+Every time the device attempts to connect, the server verifies that the device credentials are signed by the same CA that you originally installed on the server.
+
+== Choosing a provisioning method
+
+The type of provisioning that you choose depends on your requirements. If you're just testing and want to get started quickly, provisioning with shared credentials is fine. It is easier to set up but is less secure. Because server self-signs the credentials that it issues to devices, the devices have no way of verifying the integrity of the server. This level of security is not suitable for a production environment. The device should always be able to verify that is communicating with a genuine OTA Connect server. Once you move to production, you should look at some form of provisioning that uses device certificates. For more security, use a Hardware Security Module (HSM) to store the device certificates.
```

Comment:
"Because server" -> "Because the server"

Reply:
Fixed